rllib/algorithms/ppo/ppo_base_learner.py

```diff
@@ -25,17 +46,15 @@ def build(self) -> None:
             raise ValueError("lr_schedule is not supported in Learner yet")
 
         # TODO (Kourosh): We can still use mix-ins in the new design. Do we want that?
-        # Most likely not. I rather be specific about everything. kl_coeff is a
-        # none-gradient based update which we can define here and add as update with
-        # additional_update() method.
+        #  Most likely not. I rather be specific about everything. kl_coeff is a
+        #  none-gradient based update which we can define here and add as update with
+        #  additional_update() method.
 
         # We need to make sure that the kl_coeff is a framework tensor that is
         # registered as part of the graph so that upon update the graph can be updated
         # (e.g. in TF with eager tracing)
-        self.kl_coeff_val = self.hps.kl_coeff
-        self.kl_coeff = self._create_kl_variable(self.hps.kl_coeff)
-
-        self.kl_target = self.hps.kl_target
+        self.curr_kl_coeff_val = self.hps.kl_coeff
```

Comment:
Using these more descriptive names now. `curr...` means: it may change over time.